src/operator/rnn_impl.h

```diff
@@ -146,15 +146,15 @@ void LstmForwardTraining(DType* ws,
   Tensor<cpu, 3, DType> hx(hx_ptr, Shape3(total_layers, N, H));
   Tensor<cpu, 3, DType> cx(cx_ptr, Shape3(total_layers, N, H));
   const int b_size = 2 * H * 4;
```

Comment:
size_t ?

Reply:
As discussed offline: `H` represents the LSTM state size, and we are not supporting LSTM states w/dimension >= 2**32, so `b_size` should remain an `int`.

Reply:
If I understand it correctly, this is also the reason that  `hidden_size` is remaining to be `int` type, right? If so, `b_size` here, representing the total size of i2h/h2h bias of four gates, still has some overflow risks.

Reply:
@zixuanweeei thanks for your feedback. I’m happy to bump b_size up to index_t here if there are overflow concerns.